jck/jck.mk

```diff
@@ -58,3 +58,12 @@ ifndef JCK_ROOT
 endif
 
 SYSTEMTEST_RESROOT=$(TEST_RESROOT)/../../system
+
+define JCK_CMD_TEMPLATE
+export JAVA_HOME=$(TEST_JDK_HOME) $(AND_IF_SUCCESS) perl $(TEST_RESROOT)$(D)..$(D)..$(D)system$(D)stf$(D)stf.core$(D)scripts$(D)stf.pl \
```

Comment:
Since we already export `JAVA_HOME` in TKG, I do not think we need to export it here anymore.

Reply:
Makes sense. I'll remove the JAVA_HOME export and run a test after the current tests are done (which will take a little while).

Reply:
Removed export of JAVA_HOME. It works fine: Test_grinder/job/Grinder/6242/console